### DISCUSSION TITLE
"Go to definition" in the same tab

### DISCUSSION THREAD

**Discussion by FarazPatankar** [zed-industries/zed#6639] [Feature Requests]
Hey y'all, started using Zed yesterday and this one's bothering me since. Sharing the current flow below:
1. I am in my `index.tsx` file
2. I `cmd+click` on the `SearchSelect` component to `Go to definition`
3. It opens `SearchSelect.tsx` but in a new tab in the same window.

Is it possible to have the `SearchSelect.tsx` file opened in the same window replacing the `index.tsx` file?

Note: I did do a search in issues finding similar things but nothing answered this specifically.

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
